README.md

```diff
@@ -56,12 +56,11 @@ Use "dependabot [command] --help" for more information about a command.
 
 ### `dependabot update`
 
-Run the `update` subcommand with the `--dry-run` flag to
-simulate a Dependabot update job for the provided ecosystem and repo
-(without actually creating any PRs).
+Run the `update` subcommand to run a Dependabot update job for the provided ecosystem and repo.
+This does not create PRs, but outputs data that could be used to create PRs.
```

Comment:
Should we consider using a flag to do the inverse of `--dry-run`? i.e. use a `--execute` flag or something to create pull requests?

Reply:
> I would rather it be a flag that enables the PR management, rather than defaulting to it and needing to remember to add the dry-run flag

That seems right on target to me 🎯